tx/textual/valuerenderer/repeated_test.go

```diff
@@ -0,0 +1,61 @@
+package valuerenderer_test
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"testing"
+
+	"cosmossdk.io/tx/textual/valuerenderer"
+	"github.com/stretchr/testify/require"
+
+	bankv1beta1 "cosmossdk.io/api/cosmos/bank/v1beta1"
+	"cosmossdk.io/tx/textual/internal/testpb"
+	"google.golang.org/protobuf/proto"
+	"google.golang.org/protobuf/reflect/protoreflect"
+)
+
+type repeatedJsonTest struct {
+	Proto    *testpb.Baz
+	Metadata map[string]*bankv1beta1.Metadata
+	Screens  []valuerenderer.Screen
+	Parses   bool
```

Comment:
Is Parses set to false because we can't parse until other renderers are finished? If so, I propose:

```suggestion
	// TODO Remove once we finished all primitive value renderers
	// https://github.com/cosmos/cosmos-sdk/pull/13696
	// https://github.com/cosmos/cosmos-sdk/pull/13853
	Parses   bool
```